ftplugin/dart/init.lua

```diff
@@ -0,0 +1,4 @@
+require("flutter-tools.lsp").attach()
+
+vim.api.nvim_buf_set_option(0, "comments", [[sO:*\ -,mO:*\ \ ,exO:*/,s1:/*,mb:*,ex:*/,:///,://]])
```

Comment:
I generally prefer `vim.opt|o|opt_local` etc syntax as it looks "simpler" and more like a simple option setting than a function call
```suggestion
vim.opt_local.comments = [[sO:*\ -,mO:*\ \ ,exO:*/,s1:/*,mb:*,ex:*/,:///,://]]
vim.opt_local.commentstring = [[//%s]]
```

Reply:
Thanks for the comment, I was not aware of this syntax.